plugins/woocommerce/src/Internal/Admin/Settings.php

```diff
@@ -212,7 +212,10 @@ public function add_component_settings( $settings ) {
 		$settings['woocommerceTranslation'] = __( 'WooCommerce', 'woocommerce' );
 		// We may have synced orders with a now-unregistered status.
 		// E.g An extension that added statuses is now inactive or removed.
-		$settings['unregisteredOrderStatuses'] = $this->get_unregistered_order_statuses();
+		 if(PageController::is_admin_page()){
+			$settings['unregisteredOrderStatuses'] = $this->get_unregistered_order_statuses();
+		 }
```

Comment:
Also have an extra space on this line as well.

```suggestion
		}
```